src/commons.sh

```diff
@@ -1,63 +1,47 @@
 . $src_script_path/kwio.sh --source-only
 
 BASE=$HOME/p/linux-trees
-MOUNT_POINT=$HOME/p/mount
 BUILD_DIR=$BASE/build-linux
 
-QEMU_ARCH="x86_64"
-QEMU="qemu-system-${QEMU_ARCH}"
-QEMU_OPTS="-enable-kvm -smp 2 -m 1024"
-VDISK="$HOME/p/virty.qcow2"
-QEMU_MNT="/mnt/qemu"
-DEFAULT_PORT="2222"
-DEFAULT_IP="127.0.0.1"
-
 TARGET="qemu"
 
-BASHPATH=/bin/bash
-
 # Default configuration
-declare -A configurations=( ["ip"]="127.0.0.1" ["port"]="2222" )
+declare -A configurations
 
+# This function is used to show the current set up used by kworkflow.
 function show_variables()
 {
-  say "Global values:"
-
-  echo -e "\tBASE: $BASE"
-  echo -e "\tBUILD_DIR: $BUILD_DIR"
-  echo -e "\tQEMU ARCH: $QEMU_ARCH"
-  echo -e "\tQEMU COMMAND: $QEMU"
-  echo -e "\tQEMU MOUNT POINT: $QEMU_MNT"
-  echo -e "\tTARGET: $TARGET"
+  local has_local_config_path="No"
 
   check_local_configuration
```

Comment:
This line should be removed since this function doesn't exist anymore and common.sh automatic loads both global and local configuration files.

Reply:
Done